internals/core_enums.py

```diff
@@ -809,6 +809,12 @@ class SkyhookDashStatus(str, Enum):
     'rollout_plan': ROLLOUT_PLAN_DISPLAYNAMES,
   }
 
+# Valid values for FeatureEntry.ai_test_eval_run_status
+class AITestEvaluationStatus(int, Enum):
+  COMPLETE = 1
```

Comment:
Question for you: Do you plan on backfilling all feature entries with the enum? If so, you may want to have a `NONE = 0` state too.

Reply:
Yeah, I was thinking of leaving it null by default, with a similar use case you mentioned (Generate vs. Regenerate). True that `last_request_time` could also mean that. The last request time was mostly in place to clear the "in progress" status if the pipeline was terminated inelegantly